internal/bminventory/inventory.go

```diff
@@ -97,6 +97,7 @@ type Config struct {
 	AgentTimeoutStart               time.Duration     `envconfig:"AGENT_TIMEOUT_START" default:"3m"`
 	ServiceIPs                      string            `envconfig:"SERVICE_IPS" default:""`
 	DeletedUnregisteredAfter        time.Duration     `envconfig:"DELETED_UNREGISTERED_AFTER" default:"168h"`
+	DeregisterInactiveAfter         time.Duration     `envconfig:"DELETED_INACTIVE_AFTER" default:"480h"`
```

Comment:
1. Why 20 days?
2. It would be helpful to add a comment at the end of the line saying how many days it is (same for the line above).